prov/rxm/src/rxm_attr.c

```diff
@@ -62,7 +62,7 @@ struct fi_domain_attr rxm_domain_attr = {
 	.data_progress = FI_PROGRESS_AUTO,
 	.resource_mgmt = FI_RM_ENABLED,
 	.av_type = FI_AV_UNSPEC,
-	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_LOCAL,
+	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_BASIC,
```

Comment:
rxm passes the memory registration to the core provider. It supports BASIC with the help of core provider so it would require a core provider to support it.